### DISCUSSION TITLE
[Bug] Bug

### DISCUSSION THREAD

**Discussion by PingClan** [danthe1st/Mee6-bypasser#32] [Support] [answered]
**Describe the bug**
The !rank Command doesnt give the members the role. Is there a new Version from the Mee6 Bypasser?

**Comment by danthe1st** [comment]
I will have a look at it. There was a restart, yesterday.

**Comment by danthe1st** [comment]
Does it work, now?
If it does not, keep in mind the following aspects:
- The highest role of `Mee6-bypasser` (normally also named `Mee6-bypasser`) needs to be higher than the role to assign.
- Enter the command `mb!show` in order to display roles to assign. The role to assign needs to be included in order to work.

**Comment by willowwwrg** [comment]
yeah im having this issue too

**Comment by danthe1st** [comment, marked as answer]
This occurs if too many people execute `!rank` at the same time as Mee6 blocks Mee6-bypasser because of too many requests. It might work if you wait a minute and try again.

## [Edit]

This should not be a problem any more.
